src/bin/inx-chronicle/api/stardust/poi/routes.rs

```diff
@@ -159,7 +160,10 @@ fn get_valid_public_keys_for_index(mut key_ranges: Box<[MilestoneKeyRange]>, ind
         match (key_range.start, key_range.end) {
             (start, _) if start > index => break,
             (start, end) if index <= end || start == end => {
-                public_keys.insert(key_range.public_key.clone());
+                // Panic: should never fail
```

Comment:
🤔

Reply:
I looked up the error conditions...if we trust Hornet and our own database then this won't fail.

Reply:
Still, returning an error isn't difficult and it would print a better error just in case.

Reply:
But think again about what this means! We just pulled those coordinator public keys from our database and they cannot be decoded. That's not something that is going to change with another API call. If that `unwrap` panics Chronicle's db got corrupted.

Reply:
We won't shutdown either way, this is the API.

Reply:
that's a good point ... it should be a big red error in the logs but no shutdown.